virt_lightning/shell.py

```diff
@@ -60,7 +60,7 @@ def up(configuration, virt_lightning_yaml_path, context):
         # that symbol more well add to check encoding block
         status_line += "🗲{name} ".format(**host)
         print(status_line)
-        domain = hv.create_domain(**host)
+        domain = hv.create_domain(name=host["name"], distro=host["distro"])
```

Comment:
To avoid this issue you have to change create_domain signature to use `**kwargs` after defined named args.

Reply:
I agree for the `**kwargs`. I preparing a better solution.

Reply:
update:

I don't thing we need `**kwargs` for this specific case and the current patch fix a problem. So I would prefer we apply the current patch. In a second time, we will prepare a better fix. WDYT?